python/sort_number.py

```diff
@@ -0,0 +1,23 @@
+import sys
+#This function will sort a given sequence of numbers
+#if value in the list of numbers is not interger or float, program will exit
+def sort_numbers(numbers):
+    for value in numbers:
+        check_value_int = isinstance(value, int)
+        check_value_float = isinstance(value, float)
+
+        if not check_value_int and not check_value_float:
+            print("Error: " + str(value) + " is not integer or float.")
```

Comment:
I would prefer using comma in prints. With comma there is no need to do type changes or add additional spaces before and after the variable
```suggestion
            print("Error:", value, "is not integer or float.")
```